vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java

```diff
@@ -251,7 +251,7 @@ public void openRelativeUrl(String relativeURL)
      * @param comparisonRule String validation rule: "is equal to", "contains", "does not contain"
      * @param text The text of the title to compare (ex. {@code <title>}<b>'text'</b>{@code </title>})
      */
-    @Then("the page title $comparisonRule '$text'")
+    @Then("page title $comparisonRule `$text`")
```

Comment:
The changed steps are very different, please split the changes into 2 commits

Reply:
Splitted into 2 commits